internal/pkg/logging/workload.go

```diff
@@ -200,20 +205,25 @@ func (s *WorkloadClient) WriteLogEvents(opts WriteLogEventsOpts) error {
 	}
 }
 
-func (s *WorkloadClient) logStreams(taskIDs []string, includeStateMachineLogs bool) (logStreamPrefixes []string) {
+func (s *WorkloadClient) logStreams(taskIDs []string, includeStateMachineLogs bool, container string) []string {
```

Comment:
just a thought (i know this was mostly here before, and it'll probably be changed re: the above `TODO`)
```
switch {
case includeStateMachineLogs:
  return []string{fmt.Sprintf("%s/%s", wkldLogStreamPrefix, container), stateMachineLogStreamPrefix}
case len(taskIDs) == 0:
  return []string{fmt.Sprintf("%s/%s", wkldLogStreamPrefix, container)}
}

var logStreamPrefixes []string
if container == "" {
  container = s.name
}
for _, taskID := range taskIDs {
  // etc
}
```

Reply:
Done! Ya it's likely going to change after the refactor, but it's still nicer this way. I bet with the original way it's written I'll be very confused again after I'm back 😂  At least this way it'll take less time for me to regain my understanding 😂